### PR TITLE
Communications module fixes

### PR DESCRIPTION
**Pull request by ron-popov**
Fixed some path issues that rose while organizing the code base and updated the JS code to also not allow float values.

### CODE REVIEW

File: Modules/Communications/html/live.html [hunk 1]
```diff
@@ -91,8 +91,9 @@
                 var valid = true;
                 str.forEach(function(element){
                     dict[element] = document.getElementById("set"+element.toUpperCase()).value;
-                    if(parseFloat(dict[element])>1.0||parseFloat(dict[element])<-1.0){ 
-                        "values must be between -1 and 1".alert();
+                    var value = parseFloat(dict[element]);
+					if(value != 0 && value != 1.0 && value != -1.0){
```
Comment (yotam180) [inline Modules/Communications/html/live.html:95]: Can you indent those conventionally? Actually, can you just prettify this file? It looks like it was lazily formatted.

**Review by yotam180 — CHANGES_REQUESTED** [review]
I know these changes aren't really a fault of this PR. But can you please take care of them?

File: Modules/Communications/html/live.html (hunk 1 above)
Comment (ron-popov) [reply Modules/Communications/html/live.html:95]: Fine 😭

Comment (ron-popov) [reply Modules/Communications/html/live.html:95]: But if am starting to fix shai's code, this won't end here

**Comment by ron-popov** [comment]
I will close this pull request then and open another one when i am done.

File: Modules/Communications/html/live.html (hunk 1 above)
Comment (yotam180) [reply Modules/Communications/html/live.html:95]: Feel free. Youv'e got a green light. :traffic_light:

**Pull request closed without merging**